.github/workflows/helm_chart.yaml

```diff
@@ -0,0 +1,140 @@
+name: Helm chart
+
+on:
+  push:
+    branches:
+      - "**"
+    tags:
+      - "chart/**"
+
+env:
+  HELM_CHART_NAME: cloudinfo
+  HELM_CHART_PATH: "${{ github.workspace }}/charts/cloudinfo"
+  HELM_PLUGIN_CHARTMUSEUM_PUSH_VERSION: 0.9.0
+  HELM_PUSH_REPOSITORY_NAME: banzaicloud-stable
+  HELM_REPOSITORIES_JSON: '{"banzaicloud-stable": "https://kubernetes-charts.banzaicloud.com", "incubator": "https://charts.helm.sh/incubator", "stable": "https://charts.helm.sh/stable"}'
+  HELM_VERSION: 3.6.0
+
+jobs:
+  # Note: currently jobs.<job-id>.(runs-on|strategy) cannot use global '${{ env }}'.
+  #
+  # Note: currently ${{ env }} cannot define dynamically initialized values.
+  environment:
+    name: Configure environment
+    runs-on: ubuntu-latest # TODO: until static global env can be used through ${{ env }}.
+    outputs:
+      git_refname: ${{ steps.set-git-refname.outputs.git_refname }}
+      github_runner_default_os: ubuntu-latest
+      helm_push_enabled: ${{ steps.set-helm-push-enabled.outputs.helm_push_enabled }}
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set Git refname
+        id: set-git-refname
+        run: |
+          GIT_REFNAME="$(echo "${{ github.ref }}" | sed -r 's@refs/(heads|pull|tags)/@@g')"
+
+          echo "GIT_REFNAME=${GIT_REFNAME}"
+          echo ::set-output name=git_refname::${GIT_REFNAME}
+
+      - name: Set Helm push enabled
+        id: set-helm-push-enabled
+        run: |
+          HELM_PUSH_ENABLED=""
+          if [ "${{ github.event_name }}" == "push" ] && echo "${{ steps.set-git-refname.outputs.git_refname }}" | grep -E -q "^chart/${{ env.HELM_CHART_NAME }}/[0-9]+.[0-9]+.[0-9]+**"; then
+            HELM_PUSH_ENABLED=1
+          else
+            printf >&2 "Unstable chart (%s) from %s event, chart will not be pushed" "${{ steps.set-git-refname.outputs.git_refname }}" "${{ github.event_name }}"
+          fi
+
+          echo "HELM_PUSH_ENABLED=${HELM_PUSH_ENABLED}"
+          echo ::set-output name=helm_push_enabled::${HELM_PUSH_ENABLED}
+
+  helm:
+    name: Helm
+    needs:
+      - environment
+    runs-on: ${{ needs.environment.outputs.github_runner_default_os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Required for correctly populating ldflags
+          fetch-depth: 0
+
+      - name: Setup Helm
```

Comment:
There is an action for this: https://github.com/dexidp/helm-charts/blob/master/.github/workflows/ci.yaml#L19

Reply:
Fixed in 7a98e01.